File: Poin B.py/RAM.py
class RAM:
    #atribut
    capacity = ""
    price = 0
    
    
    def __init__(self, c = "", p = 0) :
        self.capacity = c
        self.price = p
        
    def setCapacity(self, c= "") :
        self.capacity = c
        
    def setPrice(self, p = 0) :
        self.price = p
        
    def getCapacity(self) :
        return self.capacity
    
    def getPrice(self) :
        return self.price
    
    def output(self) :
        print("RAM                          : " + str(self.getCapacity()))
        print("Price                        : " + str(self.getPrice()))
        
    
    
        
    

File: Poin B.java/RAM.java
public class RAM {

    //atribut
    private String capacity;
    private int price;

    //konstruktor
    public RAM(String capacity, int price) {
        this.capacity = capacity;
        this.price = price;
    }
    public RAM() {
    }


    //get set methods
    public int getPrice() {
        return price;
    }
    public String getCapacity() {
        return capacity;
    }
    public void setCapacity(String capacity) {
        this.capacity = capacity;
    }
    public void setPrice(int price) {
        this.price = price;
    }

    //output
    public void output(){
        System.out.println("RAM                     : " + this.getCapacity());
        System.out.println("Price                   : " + this.getPrice());
    }

    
    


}
